### PR TITLE
Remove call to os.getlogin()

### PR DESCRIPTION
**Pull request by laispc**
Hi,

This PR removes the use of os.getlogin() due to some conflicts it has being causing in environments such as Docker.

Thanks,

Laís

### CODE REVIEW

**Review by hhslepicka — APPROVED** [review]

File: py4syn/utils/scan.py [hunk 1]
```diff
@@ -897,11 +897,14 @@ def start(self):
                 FILE_WRITER.insertComment(SCAN_COMMENT)
 
                 try:
-                    user = os.getlogin()
-                except FileNotFoundError:
+                    print('getting user from env')
```
Comment (gabrielfedel) [inline py4syn/utils/scan.py]: Are these prints necessary? It looks like debug messages

File: py4syn/utils/scan.py [hunk 2]
```diff
@@ -897,11 +897,14 @@ def start(self):
...
                     user = os.environ.get('LOGNAME') or os.environ.get('USERNAME')
                     if not user:
+                        print('getting user from pwd')
```
Comment (gabrielfedel) [inline py4syn/utils/scan.py]: the same as above

**Review by gabrielfedel — COMMENTED** [review]
For me is great, my only suggestion is remove print messages

File: py4syn/utils/scan.py (hunk 1 above)
Comment (laispc) [reply py4syn/utils/scan.py]: > Are these prints necessary? It looks like debug messages

My bad, I forgot to delete these prints. I am pulling a review branch (#52 ). :) Thanks!

Comment (laispc) [reply py4syn/utils/scan.py]:
```suggestion
                    print('getting user from env')
                    
```

**Pull request closed without merging**